build/methane_sensor.md

```diff
@@ -0,0 +1,66 @@
+# Sensor Title
+
+Sensor and waterproof case specifically for use with the Figaro NGM2611-E13, and Adafruit DHT22.
+
+## Bill of Materials
+
+| Item                                                       | Case Qty   | Buy Qty   |  Supplier Part # / CAD link |
+| ---------------------------------------------------------- | ---------- | --------- |  -------------------------- |
+| NGM2611-E13        | 1  | 1          | https://... |
+| DHT22              | 1  | 1           | https://... |
+| 1-1/4" PVC Sch 40         | 1 | 1 | |
+| 1-1/4" PVC Sch 40 End Cap | 1 | 1 | |
+| 8 conduit cable | 7.5" |1m | digikey |
+| .003" PTFE film 6" Wide | | | |
+| 8" cable tie | | | |
+| Plastidip  | consumable | |
+| Tall round cylinder  | | | |
+| Epoxy | Consumable | | |
+| Electrical tape | | | |
+| Protoboard/stripboard | | | |
+
+
+## Schematics
+
+Wiring diagrams.
+
+### Communication protocol
+
+### Power solution
+
+## Diagrams
+
+Mechanical drawings.
+
+## Build Process
+
+### Tools & Skills
+
+Tension rod (for fume hood)
+Shower rings & clips {-> S hook}
+Wire stripper
+Wire cutter
+Soldering Iron
+Tube Cutter (the small one for pvc)
+Deburring tool (find the right sized one for 1-1/4" pvc)
+Scalpel / art knife
+Gloves
+Fume hood / well ventilated area
+
+### Parts Fabrication
+
+1. Cut PTFE film to 6" x 6" square, then cut corners to make an equilateral octagon
+2. DHT22 -> proto board (4x4) w/ JST-XH 4pin plug, 10k ohm resistor
+3. Cut cable to 7.5", strip 1" off end, attach to sensors
+4. Drill hole in center of cap ___ size
+5. Cut PVC to 2.25"
+6. Cut 1/2" ring off the end of a cap
+7. 
+
+### Assembly & Waterproofing
+
+1. Cable and sensors through the cap, electrical tape to make it snug
+2. Add 2.25" PVC section to cap, then epoxy the cable in place from the outside, and the interface between pvc and pvc cap, set to cure for __ hrs
+3. Wrap PTFE around open PVC and hold in place with cable ties / rubber bands, two through each other then near the lip and one near the end of the PTFE film while remaining entirely around it, use scalpel to cut excess film after the ziptie, leaving only ~1 mm left
+3. Use pvc cap ring to put PTFE film around end of PVC cap, then epoxy arouud the cable and the two interfaces between cap and tube cure for ___ time
+4. Use the ends of the zip ties to slowly insert and extract case from plastidip and hang to cure from two clips on tension rod for 30 min within fume hod, then dip again and set to cure for 4 hours
```

Comment:
Insert and remove at a rate of 1 min per 5 seconds.  Instead far enough to cover at least 1/2 of the PVC ring.